cmake/projects/RapidXML/hunter.cmake

```diff
@@ -0,0 +1,19 @@
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    RapidXML
+    VERSION
+    "1.13"
+    URL
+    "https://github.com/hunter-packages/rapidxml/archive/v1.13-hunter.tar.gz"
+    SHA1
+    95164691ae21bd64b4dc0608a4e6d2d27c487dea
+)
+
+hunter_pick_scheme(DEFAULT url_sha1_cmake)
```

Comment:
You should add `hunter_cacheable`, see https://docs.hunter.sh/en/latest/reference/user-modules/hunter_cacheable.html